<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWorldDialog</class>
 <widget class="QDialog" name="NewWorldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewWorldDialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Gravity</string>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X - axis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Y - axis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="gravityYEdit">
      <property name="text">
       <string>-80</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="gravityXEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>World size</string>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="worldWidthEdit">
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="worldHeightEdit">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>171</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Buffer size</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="bufferEdit">
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>World time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="timeEdit">
      <property name="text">
       <string>600</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Genetic parameters</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Population size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Selection type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="mutationBox">
      <property name="title">
       <string>Mutation probability</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Body</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Wheel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="bodyMutedit">
         <property name="text">
          <string>0,0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="wheelMutedit">
         <property name="text">
          <string>1,0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="crossoverBox">
      <property name="title">
       <string>Crossover probability</string>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>One point</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Two point</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Wheel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="onePointEdit">
         <property name="text">
          <string>1,0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="twoPointEdit">
         <property name="text">
          <string>0,0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="wheelEdit">
         <property name="text">
          <string>0,0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="popSizeEdit">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Fitness</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rank</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tournament</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewWorldDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewWorldDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
